### DISCUSSION TITLE
Error when removing a Resource's list page and it's relative index route and action buttons

### DISCUSSION THREAD

**Discussion by myregistration** [filamentphp/filament#12485] [Help] [answered]
### Package

Other

### Package Version

v3

### How can we help you?

I've removed the index route from `getPages()` and I've deleted the list page and removed any relative actions.  I also am returning null or emtpy string from the `getBreadcrumb()` in the resource and edit pages in case that refers to the index page.  I've even utilize `redirectAfterSave()` in case that would redirect to the index page.  The route for the index page is no longer in the route list.  I've cleared cache using optimize:cache, I've restarted sail, I've done a fresh migration, I've even restarted my computer.  I keep getting `Route [filament.account.resources.users.index] not defined` unless I add the index route and relative list page back.   I don't want to use the index page.  Anyone know how to make it go away?  I realize there is probably another way to resolve this like a custom page, but I would like to use the relational managers and understand why this is happening.  Thanks.

**Comment by GekkeGlennDev** [comment]
Maybe your routes are cashed? try to do: `php artisan route:clear`

If this is not the case, it maybe is because a other resource is trying to create a IndexAction to that route?

**Reply by myregistration** [reply]
php artisan optimize:clear handles routes, too, but I tried your suggestion and still get error.  I even tried composer dump-autoload, but didn't help.  Any other ideas?

**Comment by myregistration** [comment, marked as answer]
I figured out the fix, based on the answer at [https://www.answeroverflow.com/m/1152311967575457862](https://www.answeroverflow.com/m/1152311967575457862).

I had implemented the `breadcrumb()` function, not `breadcrumbs()`.  I think this should be considered an issue because any reference to an omitted page can cause issues behind the scenes and it's extremely difficult to debug.   I followed the documentation instructions for omitting pages in a Resource so it should have worked without overriding `breadcrumbs()`.
